From 49c0a74c1cf0efb9e1eb9e0ae33058ec7e63d968 Mon Sep 17 00:00:00 2001
From: Stefan Hundhammer <Stefan.Hundhammer@gmx.de>
Date: Sun, 11 May 2025 16:26:10 +0200
Subject: [PATCH] Include <boost/ref.hpp> (Closes #92)

---
 src/YQPkgHistoryDialog.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/YQPkgHistoryDialog.cc b/src/YQPkgHistoryDialog.cc
index dd63a6d..967ec9c 100644
--- a/src/YQPkgHistoryDialog.cc
+++ b/src/YQPkgHistoryDialog.cc
@@ -26,6 +26,7 @@
 #include <QSplitter>
 #include <QTreeWidget>
 
+#include <boost/ref.hpp>
 #include <zypp-core/Date.h>
 #include <zypp-core/Pathname.h>
 #include <zypp-core/Url.h>
